runner/app/pipelines/text_to_image.py

```diff
@@ -125,10 +125,11 @@ def __init__(self, model_id: str):
             self.ldm.scheduler = EulerDiscreteScheduler.from_config(
                 self.ldm.scheduler.config, timestep_spacing="trailing"
             )
-        elif ModelName.SD3_MEDIUM.value in model_id:
+        elif ModelName.SD3.value in model_id:
             self.ldm = StableDiffusion3Pipeline.from_pretrained(model_id, **kwargs).to(
                 torch_device
             )
+            kwargs["torch_dtype"] = torch.bfloat16
```

Comment:
@ad-astra-video It looks like this adjustment is aimed at enabling it to run on consumer GPUs. Have you checked for any quality degradation? If so feel free to merge 👍🏻.

Reply:
I included this because all of the examples on the model card for large and medium models use `torch_dtype.bfloat16`.